README.md

```diff
@@ -178,10 +178,10 @@ For more information about contributing to the Kubernetes documentation, see:
 | [Italian](README-it.md)    | [Ukrainian](README-uk.md)  |
 | [Japanese](README-ja.md)   | [Vietnamese](README-vi.md) |
 
-## Code of conduct
+## Code of Conduct
 
 Participation in the Kubernetes community is governed by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
 
-## Thank you
+## Vote of Thanks
```

Comment:
I prefer the original and see my [comment](https://github.com/kubernetes/website/pull/29431/files#r703961820) earlier about capitalization in headings
```suggestion
## Vote of Thanks
```